contracts/arbitrum/connectors/uniswap/v3-swap/main.sol

```diff
@@ -0,0 +1,169 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3 swap.
+ * @dev Decentralized Exchange.
+ */
+
+import { TokenInterface } from "../../../common/interfaces.sol";
+import { Events } from "./events.sol";
+import { DSMath } from "../../../common/math.sol";
+import { Basic } from "../../../common/basic.sol";
+import "./interface.sol";
+
+abstract contract UniswapResolver is DSMath, Events, Basic {
+	/**
+	 * @dev uniswap v3 Swap Router
+	 */
+	ISwapRouter constant swapRouter =
+		ISwapRouter(0xE592427A0AEce92De3Edee1F18E0157C05861564);
```

Comment:
@Richa-iitr Let's move this into `helpers.sol`?

Reply:
Okay.